ws-conn/ws-connection-handler.go

```diff
@@ -109,16 +123,24 @@ func (handler *ConnectionHandlerImpl) readDataFromConnections(inputChan chan []b
 
 // Send ping message to the connection client.
 func (*ConnectionHandlerImpl) sendPingMessage(wsConn *websocket.Conn) {
-	ticker := time.NewTicker(PingPeriod)
+	ticker := time.NewTicker(pingPeriod)
 	defer ticker.Stop()
 
 	for range ticker.C {
-		err := wsConn.WriteMessage(websocket.PingMessage, []byte{})
-		if err != nil {
-			if !IsNormalWSError(err) {
-				logrus.Errorf("Error occurs on sending ping message to ws-conn. %v", err)
-			}
-			return
+		if err := wsConn.WriteMessage(websocket.PingMessage, []byte{}); err != nil {
```

Comment:
I want to avoid this error in the log to make log clean. If connection is abnormally closed any way we print an error to the log, because connection read method fails, so it is not necessary to log one more error and pollute logs.